### PR TITLE
disable s3 online upgrade

### PR DESCRIPTION
**Pull request by PiaoLingAl**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 2848c3023da86fed43334c37609d16508cef37a6

**Comment by skylarCai** [comment]
auto build success,number:172

**Comment by skylarCai** [comment]
autotest success,number:1389

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

File: hardware/board/esp_s3_12k_kit/package.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ def_config:
   CONFIG_LFS_PAGE_NUM_PER_BLOCK: 16
   LFS_CONFIG_DEBUG: 0
   LFS_CONFIG_TRACE: 0
-  MICROPY_PY_CHANNEL_ENABLE: 1
```
Comment (skylarCai) [inline hardware/board/esp_s3_12k_kit/package.yaml:34]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**